api/v1/api_v1_types.go

```diff
@@ -86,15 +86,15 @@ type IsiSnapshot struct {
 	PctReserve    float64 `json:"pct_reserve"`
 	Schedule      string  `json:"schedule"`
 	ShadowBytes   int64   `json:"shadow_bytes"`
-	Size          int64   `json:"size"`
+	Size          uint64  `json:"size"`
```

Comment:
the only other one that the API doc says should be in createdate but if I do that, then unix time conversion won't work because it doesn't want uint64. so I am leaving that for now